app/helpers/mailer_helper.rb

```diff
@@ -181,22 +181,25 @@ def style_work_tag_metadata(tags)
   end
 
   def commenter_pseud_or_name_link(comment)
+    return style_bold(t("roles.anonymous_creator")) if comment.by_anonymous_creator?
+
     if comment.comment_owner.nil?
-      t("roles.guest_commenter_name_html", name: style_bold(comment.comment_owner_name), role: style_role(t("roles.guest_with_parens")))
-    elsif comment.by_anonymous_creator?
-      style_bold(t("roles.anonymous_creator"))
+      t("roles.commenter_name_html", name: style_bold(comment.comment_owner_name), role: style_role(t("roles.guest_with_parens")))
     else
-      style_pseud_link(comment.pseud)
+      role = comment.user.official ? t("roles.official_with_parens") : t("roles.registered_with_parens")
+      pseud_link = style_link(comment.pseud.byline, user_pseud_url(comment.user, comment.pseud))
+      t("roles.commenter_name_html", name: tag.strong(pseud_link), role: style_role(role))
```

Comment:
The link is already set to red in the [style_link](https://github.com/otwcode/otwarchive/blob/66eb406c8938e50ea7b4ee5c44616d5785cf4dfa/app/helpers/mailer_helper.rb#L8) call. If it's just about `<b>` instead of `<strong>`, then `tag.b` would be an option that avoids setting the color twice.

Reply:
Or the style_link could be replaced with a plain `link_to` + `style_bold`.

Reply:
Ah I'd missed that, thanks! I'm good with keeping it as is